guides/source/5_2_release_notes.md

```diff
@@ -184,37 +857,221 @@ Please refer to the [Changelog][active-support] for detailed changes.
+
+*   Changed default behaviour of `ActiveSupport::SecurityUtils.secure_compare`,
+    to make it not leak length information even for variable length string.
+    Renamed old `ActiveSupport::SecurityUtils.secure_compare` to
+    `fixed_length_secure_compare`, and started raising `ArgumentError` in
+    case of length mismatch of passed strings.
+    ([Pull Request](https://github.com/rails/rails/pull/24510))
+
+*   Use SHA-1 to generate non-sensitive digests, such as the ETag header.
+    ([Pull Request](https://github.com/rails/rails/pull/31289),
+    [Pull Request](https://github.com/rails/rails/pull/31651))
+
+*   `assert_changes` will always assert that the expression changes,
+    regardless of `from:` and `to:` argument combinations.
+    ([Pull Request](https://github.com/rails/rails/pull/31011))
+
+*   Add missing instrumentation for `read_multi`
+    in `ActiveSupport::Cache::Store`.
+    ([Pull Request](https://github.com/rails/rails/pull/30268))
+
+*   Support hash as first argument in `assert_difference`.
+    This allows to specify multiple numeric differences in the same assertion.
+    ([Pull Request](https://github.com/rails/rails/pull/31600))
+
+*   Return all mappings for a timezone identifier in `country_zones`.
+    ([Commit](https://github.com/rails/rails/commit/cdce6a709e1cbc98fff009effc3b1b3ce4c7e8db))
+
+*   Caching: MemCache and Redis `read_multi` and `fetch_multi` speedup.
+    Read from the local in-memory cache before consulting the backend.
+    ([Commit](https://github.com/rails/rails/commit/a2b97e4ffef971607a1be8fc7909f099b6840f36))
 
 Active Job
------------
+----------
 
 Please refer to the [Changelog][active-job] for detailed changes.
 
-### Removals
+### Notable changes
 
-ToDo
+*   Add support for compatibility with redis-rb gem for 4.0 version.
+    ([Pull Request](https://github.com/rails/rails/pull/30748))
+
+*   Allow block to be passed to `ActiveJob::Base.discard_on` to allow custom
+    handling of discard jobs.
+    ([Pull Request](https://github.com/rails/rails/pull/30622))
+
+Ruby on Rails Guides
+--------------------
+
+Please refer to the [Changelog][guides] for detailed changes.
 
 ### Notable changes
 
-ToDo
+*   Add
+    [Threading and Code Execution in Rails](threading_and_code_execution.html)
+    Guide.
+    ([Pull Request](https://github.com/rails/rails/pull/27494))
+
+*   Add [Active Storage Overview](active_storage_overview.html) Guide.
+    ([Pull Request](https://github.com/rails/rails/pull/31037))
 
 Credits
 -------
 
 See the
-[full list of contributors to Rails](http://contributors.rubyonrails.org/) for
-the many people who spent many hours making Rails, the stable and robust
+[full list of contributors to Rails](http://contributors.rubyonrails.org/)
+for the many people who spent many hours making Rails, the stable and robust
 framework it is. Kudos to all of them.
 
+[Ruby on Rails 5.2 Release Notes](5_2_release_notes.html) was compiled
+by [bogdanvlviv](http://bogdanvlviv.com).
```

Comment:
Sorry but we don't mention the person who compiled the release notes. The same way we don't mention the contributors directly in all these entries or in who wrote a guide.

Reply:
Looks like we used to add a mention who wrote a release notes guide https://github.com/rails/rails/blob/3c9edcffb60b21beb6d54a72d38aa1883fc7b2c9/guides/source/3_2_release_notes.md. We haven't added this mention since Rails 4.0 so I'm removing this.